modules/serverless-rn-1-30-0.adoc

```diff
@@ -0,0 +1,79 @@
+====
+
+[id="new-features-1-30-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.9.
+* {ServerlessProductName} now uses Knative Eventing 1.9.
+* {ServerlessProductName} now uses Kourier 1.9.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.9.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.9.
+* The `kn func` CLI plug-in now uses `func` 1.10.1.
+
+* {ServerlessProductName} now runs on HyperShift-hosted clusters.
+
+* {ServerlessProductName} now runs on single-node OpenShift.
+
+* {FunctionsProductName} nows supports {pipelines-title} versions 1.10 and 1.11. Older versions of {pipelines-title} are no longer compatible with {FunctionsProductName}.
+
+* Eventing integration with {SMProductName} is now available as a Technology Preview (TP) feature.
++
+The integration includes the following:
++
+** `PingSource`
+** `ApiServerSource`
+** Knative Source for Apache Kafka
+** Knative Broker for Apache Kafka
+** Knative Sink for Apache Kafka
+** `ContainerSource`
+** `SinkBinding`
+** `InMemoryChannel`
+** `KafkaChannel`
+** Channel-based Knative Broker
+
+* Pipelines-as-code for {FunctionsProductName} is now available as a Technology Preview (TP).
+
+* You can now configure the burst and queries per second (QPS) values for `net-kourier`.
+
+* {FunctionsProductName} users now have the ability to override the `readiness` and `liveness` probe values in the `func.yaml` file for individual Quarkus functions.
++
+See "Functions development reference guide" for guidance on Quarkus, TypeScript, and Node.js functions.
+
+* Beginning {ServerlessProductName}, Kourier controller and gateway manifests have the following limits and requests by default:
+** requests:
+*** cpu: 200m
+*** memory: 200Mi
+** limits:
+*** cpu: 500m
+*** memory: 500Mi
++
+See the "Overriding Knative Serving system deployment configurations" section of {ServerlessProductName} documentation.
+
+* The `NamespacedKafka` annotation, which was a Technology Preview (TP) feature, is now deprecated in favor of the standard Kafka broker with no data plane isolation.
+
+[id="fixed-issues-1-30_{context}"]
+== Fixed issues
+
+* Previously, the `3scale-kourier-gateway` pod was sending thousands of `net-kourier-controller` DNS queries daily. New queries were being sent for each `NXDOMAIN` reply. This continued until the correct DNS query was produced.
++
+The query now has the `net-kourier-controller.knative-serving-ingress.svc.<cluster domain>.` fully-qualified domain name (FQDN) by default, which solves the problem.
+
+[id="known-issues-1-30-0_{context}"]
+== Known issues
+
+* Building and deploying a function using Podman version 4.6 fails with the `invalid pull policy "1"` error.
++
+To workaround this issue, use Podman version 4.5.
```

Comment:
If using workaround as a verb, should it be two words?

workaround -> work around